apps/website/app/(docs)/docs/roam/pages/discourse-context-overlay.md

```
@@ -0,0 +1,42 @@
+- [Discourse Context Overlay](./discourse-context-overlay)
+- [Discourse Attributes](./discourse-attributes)
+- [Node Index](./node-index)
+
+The Discourse Context Overlay adds an icon next to each discourse node wherever it is referenced. This icon provides access to two key features:
+
+1. a popover to view the [Discourse context](./discourse-context) of the node inline, and
+2. a "Discourse Score" component that displays a customizable score for the node that is some function of its relations to other nodes in the discourse graph.
+
+The overlay is an optional feature that is turned off by default. To turn it on, go to the grammar tab in the config page and check the box for overlay.
+
+![](/docs/roam/settings-discourse-context-overlay.png)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)